sklearn/linear_model/sgd_fast.pyx

```diff
@@ -700,7 +728,7 @@ def _plain_sgd(np.ndarray[double, ndim=1, mode='c'] weights,
 
     w.reset_wscale()
 
```

Comment:
It would raise a warning for each partial_fit (which has `max_iter=1`).
Instead I suggest to raise it in the [_fit method](https://github.com/scikit-learn/scikit-learn/pull/5036/files#diff-c58bb22c5737f200d2494695f6dea33dR451).